src/windshaft/filters/collection.js

```diff
@@ -2,44 +2,39 @@ var _ = require('underscore');
 var Backbone = require('backbone');
 
 module.exports = Backbone.Collection.extend({
+
+  /**
+   * @param  {Array.<Filter>} filters Array with the filters to add to the collection.
+   * @param  {Array.<LayerModel>} layers  Array of LayerModel in the same order passed to the WindshaftMap.
+   * @returns {WindshaftFilterCollection}
+   */
   initialize: function (filters, layers) {
     this.layers = layers;
     Backbone.Collection.prototype.initialize.apply(this, filters);
   },
 
   toJSON: function () {
```

Comment:
Because output is a bit different. Now we always have an object with filters per layer (although no filter was applied on the layer). In summary, if our map has three layers without filters we get three objects. 

Before we only generate output if there were any "active filter". If there were no active filter we get an empty array. That is the only difference with current change.

If you think it is important I can change it.